NAPS2.Lib.WinForms/EtoForms/WinForms/WinFormsListView.cs

```diff
@@ -82,13 +87,38 @@ private void CustomRenderItem(object? sender, DrawListViewItemEventArgs e)
         }
         else
         {
-            height = ImageSize;
+            if (_config.Get(c => c.EnableThumbnailText))
+                height = ImageSize - 12;
+            else
+                height = ImageSize;
             width = (int) Math.Round(height * (image.Width / (double) image.Height));
         }
         var x = e.Bounds.Left + (e.Bounds.Width - width) / 2;
         var y = e.Bounds.Top + (e.Bounds.Height - height) / 2;
         e.Graphics.DrawImage(image, new Rectangle(x, y, width, height));
+        if (_config.Get(c => c.EnableThumbnailText)) 
+        { 
+            // Draw the text below the image
+            //var draw = _config.Get(c => c.EnableThumbnailText);
+            // Create string to draw.
+            String drawString = (e.ItemIndex+1).ToString() + " / "+_view.Items.Count.ToString(); 
```

Comment:
I like this way of doing it (using the indexes directly here), I was originally thinking of using the GetLabel code but this should more reliably update as the number and position of items changes.